### ISSUE TITLE
Create a Control Gallery

### ISSUE THREAD

**Issue by nsgonultas** [joaoventura/pylibui#44]
Original libui has a [control gallery](https://github.com/andlabs/libui/tree/master/examples/controlgallery) that looks really cool. It's like an example of all controls. I think we should add a control gallery to pylibui too. And we can improve it as we add new controls. What do you think?

**Comment by joaoventura** [comment]
Sounds great, go ahead! 👍

**Comment by nsgonultas** [comment]
We need a control gallery that is identical to libui's control gallery. So we can easily see which controls are missing and implement them. 

I suggest starting with compiling [libui's control gallery](https://github.com/andlabs/libui/blob/master/examples/controlgallery/main.c) and see what it looks like if you want to work on this issue. Please let us know -by commenting- if you want to pick this issue and feel free to ask for help if you need :octocat:

**Comment by superzazu** [comment]
Yeah, it's been on mind for some time. I'm interested in doing this :-)

Here's some screenshots of the current control-gallery (the ones on the libui README are not up to date):

<img width="752" alt="capture d ecran 2016-11-12 11 32 17" src="https://cloud.githubusercontent.com/assets/1141722/20237287/0285483a-a8ce-11e6-8c5c-278bcf1ad54c.png">
<img width="752" alt="capture d ecran 2016-11-12 11 32 26" src="https://cloud.githubusercontent.com/assets/1141722/20237289/07ad006e-a8ce-11e6-80ce-9c0e4f329c9c.png">
<img width="752" alt="capture d ecran 2016-11-12 11 32 28" src="https://cloud.githubusercontent.com/assets/1141722/20237290/0b92d9ce-a8ce-11e6-8e92-292746543c67.png">

**Comment by nsgonultas** [comment]
@superzazu I'm looking forward to your PR :octocat:

**Comment by superzazu** [comment]
I'm currently busy with multiple things in life, and I don't think I'll be able to do this PR soon. If anyone wants to work on this, that's okay :)

**Comment by MH15** [comment]
I've begun to build this- we're missing forms specifically.

**Comment by neroist** [comment]
I've started to make it too, missing ColorButton, FontButton, and EditableCombobox. Plus a open folder dialog.
